plugins/ca-kafka-local/README.md

```diff
@@ -0,0 +1,39 @@
+# CA Kafka Local Devbox Plugin
+
+At Culture Amp we use Kafka for sharing data between services, and the private [kafka-local](https://github.com/cultureamp/kafka-local) repo is how we run that locally.
```

Comment:
Note I've used `kafka-local` rather than `local-kafka` throughout this PR, given we're renaming soon: https://cultureamp.atlassian.net/browse/FEF-1539

Reply:
So people won't be able to use this plugin before the renaming? ;)

Reply:
It doesn't affect the plugin at all, just the docs, which I thought was an okay trade-off given we're hoping to do the rename next week.